### PR TITLE
Chats between users

### PR DESCRIPTION
**Pull request by bkarakala**

### CODE REVIEW

**Comment by SriHarikaV** [comment]
Everything looks good.

**Review by SriHarikaV — APPROVED** [review]

File: frontend/revive-gallery-app/src/__tests__/ProductForm.test.js [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-import { MemoryRouter } from "react-router-dom";
 import ProductForm from '../components/products/ProductForm';
 
 test('renders the product form', () => {
-  render(
-    <MemoryRouter> // MemoryRouter mimics BrowserRouter or HashRouter
-        <ProductForm />
-    </MemoryRouter>
-  );
+  render(<ProductForm />);
```
Comment (anjaliputta) [inline frontend/revive-gallery-app/src/__tests__/ProductForm.test.js]: @bkarakala This does not work if the component uses useNavigate hook. Please revert it back to MemoryRouter

File: frontend/revive-gallery-app/src/__tests__/ProductForm.test.js [hunk 2]
```diff
@@ -17,11 +12,7 @@ test('renders the product form', () => {
 });
 
 test('test the file formats', () => {
-  render(
-    <MemoryRouter> // MemoryRouter mimics BrowserRouter or HashRouter
-        <ProductForm />
-    </MemoryRouter>
-  );
+  render(<ProductForm />);
```
Comment (anjaliputta) [inline frontend/revive-gallery-app/src/__tests__/ProductForm.test.js]: @bkarakala This does not work if the component uses useNavigate hook. Please revert it back to MemoryRouter.
Apply the same for line 48.

File: frontend/revive-gallery-app/src/components/home/Navbar.js [hunk 3]
```diff
@@ -24,53 +24,33 @@ const Navbar = () => {
...
-                </li>
-              </ul>
-            )}
-          </li>
-        ): 
-        (<li>
-          <Link to="/login">Sign in/</Link>
-          <Link to="/register">Sign up</Link>
-        </li>)
-        }
```
Comment (anjaliputta) [inline frontend/revive-gallery-app/src/components/home/Navbar.js]: @bkarakala I think we can leave these as they are; we do need to remove them. Even though those links do not provide meaningful data as of now, they do not break the code.

File: frontend/revive-gallery-app/src/components/products/ProductDetails.js [hunk 4]
```diff
@@ -28,33 +31,48 @@ const ProductDetails = () => {
...
+  const productDemo = {
+    title: 'Sample Product',
+    description: 'This is a test product description.',
+    images: [watch1, watch2, watch3, watch4, watch5],
+  };
+ 
+  const categories = [
+    { id: 1, name: 'Category A' },
+    { id: 2, name: 'Category B' },
+  ];
```
Comment (anjaliputta) [inline frontend/revive-gallery-app/src/components/products/ProductDetails.js]: @bkarakala we can remove this hardcoded data as we are using real time data from db.

**Review by anjaliputta — APPROVED** [review]

**Pull request merged**